.github/workflows/publish-new-release.yml

```diff
@@ -249,3 +249,14 @@ jobs:
     with:
       downstream-version: ${{ needs.release-version.outputs.RELEASE_VERSION }}
     secrets: inherit
+
+  publish-docusaurus:
+    name: Publish docusaurus docs
+    permissions:
+      contents: write
+      packages: write
+    needs: [ secret-presence, publish-to-swaggerhub ]
```

Comment:
haven't tried this, but you probably want to also add the `github-release` job to the `needs` list, otherwise you could potentially reference a `downstream-version`, for which no GH Release exists yet.